src/pages/index.js

```diff
@@ -1,13 +1,19 @@
-import React from "react";
-import Header from "../components/header";
+import React, { Suspense } from "react";
+import Header from "../components/Header";
+import "../components/i18n";
+import { useTranslation } from "react-i18next";
+import '@trussworks/react-uswds/lib/uswds.css'
+import '@trussworks/react-uswds/lib/index.css';
 
-const Home = () => {
+function Home() {
+  const { t, i18n } = useTranslation();
   return (
-    <div>
-      <Header />
-      <h1>Hello World!</h1>
-    </div>
+    <Suspense fallback="loading">
+      <div>
+        <Header t={t} i18n={i18n} />
```

Comment:
I wonder if we should use the I18nextProvider instead of passing as props: https://react.i18next.com/latest/i18nextprovider

Reply:
Great point, used i18nextProvider instead.